src/mozillians.js

```diff
@@ -5,13 +5,17 @@ import url from 'url'
 import _ from 'lodash'
 import crypto from 'crypto'
 
-/** Transient HTTP errors that causes a retry */
+/** Transient error.codes that cause a retry */
```

Comment:
Please don't mix http codes and node.js error code...
My suggestion is that we always retry if it's a 5xx :)